### DISCUSSION TITLE
How do I include large binary files in the source packages when making a Release?

### DISCUSSION THREAD

**Discussion by cqcallaw** [community/community#23156] [New to GitHub]
<p>My project includes large binary files versioned with <a href="https://git-lfs.github.com/" rel="noopener nofollow ugc">git-lfs</a> that must be present to build the project. When I generate a new <a href="https://docs.github.com/en/github/administering-a-repository/about-releases" rel="noopener nofollow ugc">release</a>, the GitHub UI auto-generates a source archive of the repository at the given revision, but doesn’t include the git-lfs versioned files.</p>
<p>How do I configure the Release interface to include git-lfs files in the auto-generated source archives? Alternatively, is there a way to disable the auto-generation of the source archives?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Do you have your repository set to include lfs objects in archives?</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/administering-a-repository/managing-git-lfs-objects-in-archives-of-your-repository#about-git-lfs-objects-in-archives" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/administering-a-repository/managing-git-lfs-objects-in-archives-of-your-repository#about-git-lfs-objects-in-archives" target="_blank" rel="noopener">Managing Git LFS objects in archives of your repository - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/administering-a-repository/managing-git-lfs-objects-in-archives-of-your-repository#about-git-lfs-objects-in-archives" target="_blank" rel="noopener">//docs.github.com/en/github/administering-a-repository/managing-git-lfs-objects-in-archives-of-your-repository</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by cqcallaw** [comment]
<p>Wow, I definitely searched for that, and definitely also missed it. Thanks for the tip.</p>
<p>With that option enabled, the tarball contains the large files, but the zip file does not. Is this expected behavior?</p>

**Comment by canuckjacq** [comment]
<p>No, that isn’t expected behaviour, although I’ve seen it before where there was a caching issue. Can you try in an Incognito or Private Browsing window?</p>

**Comment by cqcallaw** [comment]
<p>Yep, that did the trick. Thanks for your help!</p>
